packages/web-components/src/components/cbp-structured-list/cbp-structured-list.scss

```diff
@@ -67,8 +67,8 @@ cbp-structured-list {
   display: block; 
   background-color: var(--cbp-structured-list-color-background);
   color: var(--cbp-structured-list-color);
-  
-  [slot] {
+
+  & > [slot] { 
```

Comment:
If this is only targeting the header, target the specific slot name rather than using > for specificity. It's more clear to read and I wasn't clear at first glance why this wasn't targeting the footer as well.

Reply:
update as per call